tests/integration/ceph_upgrade_test.go

```diff
@@ -98,59 +116,80 @@ func (s *UpgradeSuite) TestUpgradeToMaster() {
 	objectStoreName := "upgraded-object"
 	runObjectE2ETestLite(s.helper, s.k8sh, s.Suite, s.namespace, objectStoreName, 1)
 
-	// verify that we're actually running 0.8 before the upgrade
+	// verify that we're actually running the right pre-upgrade image
 	operatorContainer := "rook-ceph-operator"
 	version, err := k8sutil.GetDeploymentImage(s.k8sh.Clientset, systemNamespace, operatorContainer, operatorContainer)
-	assert.Nil(s.T(), err)
+	assert.NoError(s.T(), err)
 	assert.Equal(s.T(), "rook/ceph:"+installer.Version1_0, version)
 
 	message := "my simple message"
 	preFilename := "pre-upgrade-file"
-	assert.Nil(s.T(), s.k8sh.WriteToPod("", podName, preFilename, message))
-	assert.Nil(s.T(), s.k8sh.ReadFromPod("", podName, preFilename, message))
-	assert.Nil(s.T(), s.k8sh.WriteToPod(s.namespace, filePodName, preFilename, message))
-	assert.Nil(s.T(), s.k8sh.ReadFromPod(s.namespace, filePodName, preFilename, message))
+	assert.NoError(s.T(), s.k8sh.WriteToPod("", podName, preFilename, message))
+	assert.NoError(s.T(), s.k8sh.ReadFromPod("", podName, preFilename, message))
+	assert.NoError(s.T(), s.k8sh.WriteToPod(s.namespace, filePodName, preFilename, message))
+	assert.NoError(s.T(), s.k8sh.ReadFromPod(s.namespace, filePodName, preFilename, message))
+
+	// Gather logs before Ceph upgrade to help with debugging
+	s.k8sh.PrintPodDescribe(s.namespace)
```

Comment:
Can we remove this verbose info before merge?

Reply:
I would rather not. It is a lot of info for cases where there is a success, but when things go wrong, there is nothing to help debug what is wrong in between when the cluster is first installed and when the cluster is done upgrading.

Reply:
What if you called `CollectOperatorLog()` here? Or you really need all the pod describe? It's just really verbose in the log to print this with every test run before and after the upgrade. What if you checked for `Env.Logs != "all"` before printing it, similar to how the logs are collected if "all"? Then you could add `[all logs]` to the PR if you want to print this.